lib/pure/times.nim

```diff
@@ -219,17 +219,20 @@ proc getDayOfYear*(monthday: MonthdayRange, month: Month, year: int): YeardayRan
   else:
     result = daysUntilMonth[month] + monthday - 1
 
-proc getDayOfWeek*(monthday: MonthdayRange, month: Month, year: int): WeekDay {.tags: [], raises: [], benign .} =
-  ## Returns the day of the week enum from day, month and year.
-  ## Equivalent with ``initDateTime(day, month, year).weekday``.
-  assertValidDate monthday, month, year
-  # 1970-01-01 is a Thursday, we adjust to the previous Monday
-  let days = toEpochday(monthday, month, year) - 3
+proc getDayOfWeek*(epochday: int64): WeekDay {.tags: [], raises: [], benign .} =
+  ## Returns the day of the week enum from epoch day
+  let days = epochday - 3
   let weeks = (if days >= 0: days else: days - 6) div 7
   let wd = days - weeks * 7
   # The value of d is 0 for a Sunday, 1 for a Monday, 2 for a Tuesday, etc.
   # so we must correct for the WeekDay type.
   result = if wd == 0: dSun else: WeekDay(wd - 1)
+    
+proc getDayOfWeek*(monthday: MonthdayRange, month: Month, year: int): WeekDay {.tags: [], raises: [], benign .} =
+  ## Returns the day of the week enum from day, month and year.
+  ## Equivalent with ``initDateTime(day, month, year).weekday``.
```

Comment:
I know it was there before but I think correct English is  "equivalent to".

Reply:
yep, all done